integrations/GoogleOptimize/browser.js

```diff
@@ -0,0 +1,49 @@
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+
+class GoogleOptimize {
+  constructor(config) {
+    this.name = "GOOGLE_OPTIMIZE";
+    this.trackingId = config.trackingId;
+    this.containerId = config.containerId;
+    this.async = config.async;
+  }
+
+  init() {
+    logger.debug("===in init Google Optimize===");
+    if (!this.trackingId) {
+      return;
+    }
+    ScriptLoader(
+      "Google Tag Manager",
+      `https://www.googletagmanager.com/gtag/js?id=${this.trackingId}`
+    );
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag("js", new Date());
+    gtag("config", `${this.trackingId}`);
```

Comment:
Why are we doing this (option 3) when we are already loading the async/sync script below (option 2)?

Reply:
This is for Google anaylytics , it is always required. Google Optimize can only process results if data is received from Analytics

Reply:
This is not ideal. We should come up with a plan for it.

Reply:
In Google optimize its written that without Analytics it won't work. Since data used by Optimize is fetched by Analytics only.

Reply:
Let's add this as a setting in the config as discussed